canonical-web-and-design/snapcraft.io
# Fsf improvements

## Done

- Updated all snapcraft.yaml's so they build

## Issue / Card

Fixes #2381

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit http://0.0.0.0:8004/first-snap
- Follow all the tutorials to confirm they work (this will take some time)
